package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "cuttle",
   "description": "The deepest card game under the sea",
-  "version": "4.0.7",
+  "version": "4.1.7",
```

The patch version resets to 0 when we bump a minor version
```suggestion
  "version": "4.1.0",
```

---

Ah right, silly mistake